dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/event/data/JdbcEnrollmentAnalyticsManager.java

```diff
@@ -643,7 +643,19 @@ private Condition buildInFilterCondition(
   private Condition buildStandardFilterCondition(
       QueryFilter filter, QueryItem item, CteDefinition cteDef) {
     String value = getSqlFilterValue(filter, item);
-    String operator = "NULL".equals(value) ? "is" : filter.getSqlOperator();
+    String operator = filter.getSqlOperator();
+
+    if (value.trim().equalsIgnoreCase("null")) {
+      if (filter.getOperator() == QueryOperator.EQ || filter.getOperator() == QueryOperator.IEQ) {
+        operator = "is";
+      } else if (filter.getOperator() == QueryOperator.NEQ
```

Comment:
There is now a method in master `isNotEqualTo()` for `QueryOperator` which we can use here.

https://github.com/dhis2/dhis2-core/blob/master/dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/QueryOperator.java#L101